assets/model-diagram.svg

```diff
@@ -0,0 +1,50 @@
+<svg xmlns="http://www.w3.org/2000/svg" aria-labelledby="chart-title-mermaid-1669067354847 chart-desc-mermaid-1669067354847" style="max-width:396.227px;background-color:#fff;font-family:&quot;trebuchet ms&quot;,verdana,arial,sans-serif;font-size:16px;fill:#333" viewBox="0 0 396.227 815">
```

Comment:
This image is unreadably small when viewed standalone. https://raw.githubusercontent.com/whatwg/storage/651b26d44e2572123dbc82f93adb61359ed71dda/assets/model-diagram.svg

Will it be extremely small when embedded in the spec, too?

Reply:
I'm not quite sure what's happening, but on Firefox it seems it gets smaller with larger viewport and gets bigger with smaller-but-bigger-than-the-image viewport. Anyway we set `width` and `height` manually in HTML, so it should be fine.